front/app/modules/commercial/matomo/matchPath.ts

```diff
@@ -29,23 +35,19 @@ function compilePath(path, options) {
 /**
  * Public API for matching a URL pathname to a path.
  */
-function matchPath(pathname, options = {} as any) {
-  if (typeof options === 'string' || Array.isArray(options)) {
-    // eslint-disable-next-line no-param-reassign
-    options = { path: options };
-  }
-
-  const { path, exact = false, strict = false, sensitive = false } = options;
-
-  const paths = [].concat(path);
```

Comment:
A bit of a weird operation. I just renamed the parameter of matchPath directly to paths and I don't do this concatenation that doesn't seem to do anything.